docs/cli.md

```diff
@@ -332,54 +332,54 @@ done
 #### Command options
 
 <!-- prettier-ignore -->
-|        command option     |       description      | positional arguments |
-| ----------- | ----------- | ----------- |
-| `export`      | export data in a specified format to a specified location  | - |
-| `database configure`      | create a new database user for the Viam organization's MongoDB Atlas Data Federation instance, or change the password of an existing user. See [Configure data query](/data/query/#configure-data-query)  | - |
-| `database hostname`      | get the MongoDB Atlas Data Federation instance hostname and connection URI. See [Configure data query](/data/query/#configure-data-query)  | - |
-| `dataset add`      | add a new image to an existing dataset by its file id, or add a group of images by specifying a filter | `filter` |
-| `dataset remove`      | remove an existing image from a dataset by its file id, or remove a group of images by specifying a filter | `filter` |
-| `delete binary`      | delete binary data  | - |
-| `delete tabular`      | delete tabular data  | - |
-| `--help`      | return help      | - |
+| Command option | Description | Positional arguments |
+| -------------- | ----------- | -------------------- |
+| `export`       | Export data in a specified format to a specified location  | - |
+| `database configure`      | Create a new database user for the Viam organization's MongoDB Atlas Data Federation instance, or change the password of an existing user. See [Configure data query](/data/query/#configure-data-query)  | - |
+| `database hostname`      | Get the MongoDB Atlas Data Federation instance hostname and connection URI. See [Configure data query](/data/query/#configure-data-query)  | - |
+| `dataset add`      | Add a new image to an existing dataset by its file id, or add a group of images by specifying a filter | `filter` |
+| `dataset remove`      | Remove an existing image from a dataset by its file id, or remove a group of images by specifying a filter | `filter` |
+| `delete binary`      | Delete binary data  | - |
+| `delete tabular`      | Delete tabular data  | - |
+| `--help`      | Return help      | - |
 
 ##### Positional arguments: `dataset`
 
 <!-- prettier-ignore -->
-| argument | description |
-| ----------- | ----------- | ----------- |
+| Argument | Description |
+| ----------- | ----------- |
 | `filter`     | `add` or `delete` images from a dataset using a filter. See [Using the `filter` argument)](#using-the-filter-argument).|
 | `ids`     | `add` or `delete` images from a dataset by specifying one or more file ids as a comma-separated list. See [Using the `ids` argument)](#using-the-ids-argument).|
-| `--help`      | return help |
+| `--help`      | Return help |
 
 ##### Named arguments
 
 <!-- prettier-ignore -->
-|        argument     |       description | applicable commands | required |
+| Argument     | Description | Applicable commands | Inclusion |
```

Comment:
`Inclusion` seems a bit odd to me here: it doesn't really help me to understand what to do with the column contents. You mean: I can find out whether I _must_ include or _may_ include the listed argument. Would it be more direct to state more directly? The user is already likely looking for the words 'optional' or 'required' in this context.

```suggestion
| Argument     | Description | Applicable commands | Required |
```

Or:

```suggestion
| Argument     | Description | Applicable commands | Required / optional |
```

Reply:
I was just following what we do throughout the rest of the docs for consistency, and because I find clearly putting "required" or "optional" as much better than "true" and "false." However, I've always thought "Inclusion" seems a bit awk so maybe we should separately revisit for all the docs??

Reply:
Oh my goodness, you're right it's everywhere. Nevermind: let's definitely stick with the standard here, and let's consider as a team an alternative for down the line.

This LGTM!